### PR TITLE
Add initial branch outline

### PR DESCRIPTION
**Pull request by irmakerdem**
E & J, see the branch outline and comment on what you think should be added!

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// chores/0-initial-page-layout
+// feature/0-initial-page-layout
```
Comment (emmacbd) [inline scripts.js:2]: HTML and CSS?

File: scripts.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+// feature/1-create-classes
+// feature/2-show-random-palette
+// feature/3-save-palette
```
Comment (jordan-sullivan) [inline scripts.js:5]: Love this one! -J

File: scripts.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+// feature/4-lock-colors
```
Comment (emmacbd) [inline scripts.js:6]: don't forget lock icon

**Review by emmacbd — CHANGES_REQUESTED** [review]

**Review by jordan-sullivan — APPROVED** [review]
Looks good to me unless @emmacbd  has changes! -J

File: scripts.js [hunk 4]
```diff
@@ -6,3 +6,4 @@
 // feature/4-lock-colors
 // feature/5-delete-saved-palette
 // feature/initial-readme
+// extensions if we have time
```
Comment (irmakerdem) [inline scripts.js:9]: E, i added everything!

**Comment by jordan-sullivan** [comment]
Great start to the project! @emmacbd and @irmakerdem

**Pull request merged**